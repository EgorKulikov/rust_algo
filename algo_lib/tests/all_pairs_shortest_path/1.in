4 3 4
0 1 2
1 2 2
3 3 1
0 2
1 2
3 0
3 3
2 1 2
0 1 100
0 1
1 0
0 0 0
