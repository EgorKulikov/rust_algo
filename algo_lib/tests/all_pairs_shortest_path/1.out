4
2
Impossible
0

100
Impossible
